pom.xml

```diff
@@ -30,7 +30,7 @@
 
         <annotations.version>23.0.0</annotations.version>
         <ant.version>1.10.12</ant.version>
-        <apollo.version>3.0.0</apollo.version>
+        <apollo.version>3.8.0</apollo.version>
```

Comment:
We have a new release https://github.com/apollographql/apollo-kotlin/releases/tag/v3.8.1

Reply:
Updated to `3.8.1`!